Jenkinsfile_nightly

```diff
@@ -77,4 +77,22 @@ withNightlyPipeline(type, product, component) {
   loadVaultSecrets(secrets)
   enableFullFunctionalTest()
   enableSlackNotifications('#darts-builds')
+
+//  before('fullFunctionalTest') {
+//    // set the tags to run on the nightly build
+//    env.INCLUDE_TAGS = "This | That | Other"
+//  }
```

Comment:
This can and should be configured to run tags covering all completed tests, so that they all run on the nightly build.